src/XRP/network-clients/web-socket-network-client.ts

```diff
@@ -159,6 +165,53 @@ export default class WebSocketNetworkClient {
     return response as WebSocketStatusResponse
   }
 
+  /**
+   * Submits an account_lines request to the rippled Web Socket API.
+   * @see https://xrpl.org/account_lines.html
+   *
+   * @param account The XRPL account to query for trust lines.
+   */
+  public async getAccountLines(
+    account: string,
+    peerAccount?: string,
+  ): Promise<WebSocketAccountLinesResponse> {
+    const accountLinesRequest = {
+      id: 'account_lines_' + account,
+      command: 'account_lines',
+      account,
+      ledger_index: 'validated',
+      peer: peerAccount,
+    }
+    const accountLinesResponse = await this.sendApiRequest(accountLinesRequest)
+    return accountLinesResponse as WebSocketAccountLinesResponse
+  }
+
+  /**
+   * Submits a gateway_balances request to the rippled WebSocket API.
+   * @see https://xrpl.org/gateway_balances.html
+   *
+   * @param account The XRPL account for which to retrieve balances.
+   * @param addressesToExclude (Optional) An array of operational address to exclude from the balances issued.
+   * @see https://xrpl.org/issuing-and-operational-addresses.html
+   */
+  public async getGatewayBalances(
+    account: string,
+    addressesToExclude?: Array<string>,
+  ): Promise<WebSocketGatewayBalancesResponse> {
+    const gatewayBalancesRequest = {
+      id: 'gateway_balances_' + account,
```

Comment:
what if multiple requests for gateway balances for the same account are made in a short window?  do we need an additional piece of identifying information here?  Oh... I guess they won't be because the actual implementation of this on the issued currency client is going to block until the response comes back... so even if there are successive calls to `getGatewayBalances` using the issued currency client, we'll essentially always be dealing with just one thing in the waiting queue anyway.  Tell me if I'm wrong but I think you're good

Reply:
I was thinking about adding an extra number that keeps increasing to the end of the IDs here to avoid that problem - lmk if you think that's worth

Reply:
you'll probably want to do that eventually - might be worth tracking with a task right now and leaving as is for now

Reply:
Just did it now